Could not pass event InventoryOpenEvent to AntiIllegals v1.0.41

```
Could not pass event InventoryOpenEvent to AntiIllegals v1.0.41
org.bukkit.event.EventException: null
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at org.bukkit.craftbukkit.v1_12_R1.event.CraftEventFactory.callInventoryOpenEvent(CraftEventFactory.java:825) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.EntityPlayer.openContainer(EntityPlayer.java:858) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.BlockChest.interact(BlockChest.java:308) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.PlayerInteractManager.a(PlayerInteractManager.java:550) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1025) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_352]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_352]
        at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:851) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:427) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:775) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_352]
Caused by: java.lang.IllegalArgumentException: Specified enchantment cannot be applied to this itemstack
        at org.bukkit.inventory.ItemStack.addEnchantment(ItemStack.java:444) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        at org.zeroBzeroT.antiillegals.AntiIllegals.checkItemStack(AntiIllegals.java:309) ~[?:?]
        at org.zeroBzeroT.antiillegals.AntiIllegals.checkInventory(AntiIllegals.java:114) ~[?:?]
        at org.zeroBzeroT.antiillegals.AntiIllegals.checkItemStack(AntiIllegals.java:322) ~[?:?]
        at org.zeroBzeroT.antiillegals.AntiIllegals.checkInventory(AntiIllegals.java:114) ~[?:?]
        at org.zeroBzeroT.antiillegals.AntiIllegals.checkInventory(AntiIllegals.java:94) ~[?:?]
        at org.zeroBzeroT.antiillegals.Events.onInventoryOpen(Events.java:210) ~[?:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor3.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Dionysus-"42ce9ca"]
        ... 19 more
```

--- a/src/main/java/org/zeroBzeroT/antiillegals/AntiIllegals.java
+++ b/src/main/java/org/zeroBzeroT/antiillegals/AntiIllegals.java
@@ -306,7 +306,7 @@ public class AntiIllegals extends JavaPlugin {
 
                     wasFixed = true;
                     itemStack.removeEnchantment(enchantment);
-                    itemStack.addEnchantment(enchantment, enchantment.getMaxLevel());
+                    itemStack.addUnsafeEnchantment(enchantment, enchantment.getMaxLevel());
                 }
             }
         }

